lisa/commands.py

```diff
@@ -54,12 +55,14 @@ def run(args: Namespace) -> int:
 
 # check runbook
 def check(args: Namespace) -> int:
-    load_runbook(args.runbook, args.variables)
+    builder = RunbookBuilder.build_from_path(args.runbook, args.variables)
+    builder.runbook
```

Comment:
I mean what's the function of the line above?

Reply:
good catch, removed variable from the check method.